zerver/lib/actions.py

```diff
@@ -570,13 +580,13 @@ def revoke_preregistration_users(
     # actually used for analytics.
     if used_preregistration_user is not None:
         PreregistrationUser.objects.filter(
-            email__iexact=created_user_profile.delivery_email, realm=created_user_profile.realm
+            email__iexact=created_user.delivery_email, realm=created_user.realm
```

Comment:
We don't have full consistency in `user_profile` vs `user` for variable names, but this argument name change feels mostly like incidental churn.

Reply:
Hm; I take that back a little, what with the name of the column.

Reply:
@timabbott mentioned that we are gradually adapting `user` without `_profile`, so this change here is used to reflect that. Possibly we will unify the naming convention for that in the future.